pkg/flow/flow_http.go

```diff
@@ -85,49 +44,6 @@ func (f *Flow) ComponentHandler() http.HandlerFunc {
 	}
 }
 
-// ScopeHandler returns an http.HandlerFunc which will render the scope used
-// for variable references throughout River expressions.
-func (f *Flow) ScopeHandler() http.HandlerFunc {
-	return func(w http.ResponseWriter, _ *http.Request) {
-		be := builder.NewExpr()
-		be.SetValue(f.loader.Variables())
-
-		var buf bytes.Buffer
-		_, err := be.WriteTo(&buf)
-		if err != nil {
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-		} else {
-			_, _ = io.Copy(w, &buf)
-		}
-	}
-}
-
-// configBytes dumps the current state of the flow config as River.
-func (f *Flow) configBytes(w io.Writer, debugInfo bool) (n int64, err error) {
-	file := builder.NewFile()
```

Comment:
It looks like we could remove the pkg/river/token/builder package since it's not being used, but I'd like to keep that in for now since it serves as a good reference for how to work with River values.